packages/bierzo-wallet/src/routes/balance/components/index.tsx

```diff
@@ -9,13 +8,14 @@ import Tooltip from 'medulas-react-components/lib/components/Tooltip';
 import Typography from 'medulas-react-components/lib/components/Typography';
 import React from 'react';
 
+import { BalanceState } from '../../../store/balances';
 import { amountToString, trimAmount } from '../../../utils/balances';
 import receive from '../assets/transactionReceive.svg';
 import send from '../assets/transactionSend.svg';
 
 interface Props {
   readonly name: string | undefined;
-  readonly tokens: ReadonlyArray<Amount>;
+  readonly balance: BalanceState;
```

Comment:
Do we want this? I know they are the same type, but maybe we can have the redux type and do not change the balance attr here in this component. 

I suggest to write the type instead of reusing the redux one. What do you think? In this way, we keep separated components.

Reply:
Sorry do not understand your suggestion. Do you want me to create something like:
```
export interface BalanceType {
  [token: string]: Amount;
}
```
And use it as prop type? Or do you want to me cast (convert) `BalanceState` to `ReadonlyArray<Amount>` ?

Reply:
Just use: `tokens: [token: string]: Amount;` :)